lib/buffer.js

```diff
@@ -807,7 +807,7 @@ Buffer.prototype.toJSON = function() {
 
 
 function adjustOffset(offset, length) {
-  offset = +offset;
+  offset = ~~offset;
```

Comment:
It looks like `| 0` is used more frequently than `~~` throughout the `lib/` code.